python/hyperon/ext.py

```diff
@@ -0,0 +1,11 @@
+def add_atoms(func):
```

Comment:
`add_atom` can be confused with adding atoms into atomspace. May be some other name, for example: `add_atom_tokens`?

Reply:
Or may be 'register_atom_tokens` instead of `add_atoms` and `register_tokens` instead of `add_tokens`?

Reply:
I agree that `add_atom` is somewhat confusing. However, it is not the problem of this PR. It simply uses naming in the runner class. I agree that it makes sense to rename it, but it is used in too many tests, so it is a separate refactoring.

Reply:
Yes, I agree this is not the problem of this PR. Just thought it is fixed before adding import because I added such comment on previous PR.

Reply:
1) It has not yet been implemented (see the initial comment on the PR).
2) It is not clear for me if this should really be done by `ext` module. As I mentioned before, I expect this module to be used to extend `MeTTa` in Python. Importing expressions is already done by `import`. This is not the main functionality of `ext`, and I'm not sure it should be duplicated. Of course, if it appears to be convenient for users, I'll be happy to add some decarators to `ext` to wrap up `import` or to add expressions. Once again, `extend-py` extends the MeTTa runner, so if we include an additional argument to it, I'm not sure if it should be space or runner. Thus, type signatures can remain different for `extend` and `import` even if we add functionality to insert MeTTa code in the runner's space by `extend`. Having similar names (`import!`, `import-py!`) with different type signatures might be confusing. For now, I'm not sure if we should extend only the current runner (but for possibly a different space), or we should extend another runner (adding tokens to a tokenizer shouldn't be global?). I didn't test if another runner with its own tokenizer is created by `import` right now, and how `extend` within `import`'ed file will work...
So, I still prefer to keep different naming for now.

Reply:
`add_[atom, token]` is renamed